frontend/i18next.scanner/fr/translation.json

```diff
@@ -595,15 +595,15 @@
   "Filter visible contents": "Filtrer les contenus visibles",
   "There are no spaces that matches your filter": "Aucun espace ne correspond à votre filtre",
   "Filter users": "Filtrer les utilisateurs",
-  "Email notification:": "Notification email :",
+  "Email notification:": "Notification e-mail :",
   "Daily": "Quotidien",
   "None": "Aucun",
   "One mail will be sent for each event of the space.": "Un email va être envoyé pour chaque événement de cet espace.",
-  "Use this setting for spaces where you need to be kept informed of everything that happens.": "Utilisez cette option pour les espaces où vous devez rester informer de tout ce qui se passe.",
+  "Use this setting for spaces where you need to be kept informed of everything that happens.": "Utilisez cette option pour les espaces où vous devez rester informé de tout ce qui se passe.",
```

Comment:
On ne devrait pas parlé d'action comme la traduction en dessous ?

Reply:
this is an old translation key that is outdated since this PR. I corrected the historical typo when I saw it while working on this feature but it has then been replaced.

The translation key is still there even if not used anymore because the translation generation script has issues and, currently, it is safer to keep old translation keys.

I'd suggest to ignore this translation key